test/http_memprof/http_memprof.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// this code is a utility to monitor an already running mixer instance, and such
+// does not run with regular tests, so disable deadcode warning on it.
+//nolint:deadcode
```

Comment:
Curious what gives "deadcode" lint? This folder is similar to the existing "examples" folder?

Reply:
Yes, this is similar to "examples", but this code has functionality so I did not think that "examples" was the correct location. I will try again if the "deadcode" lint is still raised, however, I recall main() being detected as deadcode, which meant all of the file was deadcode since it is not called from anywhere else.

Reply:
Test passed; removing nolint:deadcode still works